datascience-spring-2016-master/src/main/java/edu/csula/datascience/weatherApi/SourceData.java

```diff
@@ -0,0 +1,119 @@
+package edu.csula.datascience.weatherApi;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.charset.Charset;
+import java.util.Collection;
+import java.util.List;
+
+import org.json.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.mongodb.util.JSON;
+
+import edu.csula.datascience.acquisition.Source;
+import twitter4j.Status;
+
+public class SourceData implements Source<JSONObject>{
+	
+	private long minId;
+    
+	
+	 public SourceData(long minId) {
+	        this.minId = minId;  
+	    }
+	 
+	
+	@Override
+	public boolean hasNext() {
+		
+		return minId > 0;
+	}
+	
+	@Override
+	public Collection<JSONObject> next() {
+		 List<JSONObject> list = Lists.newArrayList();
+		 
+		 String id="";
+		 String st = String.valueOf(minId);
+		 int len = st.length();
+		 	if(len == 1) id = "0000"+st;
+			if(len == 2) id = "000"+st;
+			if(len == 3) id = "00"+st;
+			if(len == 4) id = "0"+st;
+			if(len == 5) id = st;
```

Comment:
You can use `String.format` to format your id which is simpler than your manually doing left padding on the number yourself.